GNUmakefile

```diff
@@ -52,5 +52,16 @@ serialize:
 		go run . --path "api" --overrides "overrides" --mode "serialization" > temp.serial;\
 		mv -f temp.serial serialization.go;\
 
+upgrade-dcl:
+	cd tpgtools && \
+		go mod edit -require=github.com/GoogleCloudPlatform/declarative-resource-client-library@latest &&\
```

Comment:
What does this do vs running `go get github.com/GoogleCloudPlatform/declarative-resource-client-library@latest`? Will it just avoid getting the new version on the user's machine?

Reply:
Yep, that's all!

Reply:
Oh, also it doesn't depend on what version of go you're running or your setting of GO111MODULES (or whatever that env var is called)